roles/grafana/tasks/graphite.yml

```diff
@@ -20,34 +20,38 @@
 
 - name: enable services
   service:
-      name: carbon-cache
-      state: started
-      enabled: yes
+    name: carbon-cache
+    state: started
+    enabled: yes
   when: manage_services|default(false)
 
 - name: tweak httpd config
   replace:
-      dest: '{{ httpd_config_parts_dir }}/15-graphite-web.conf'
-      regexp: "Require local$"
-      replace: "Require all granted"
+    dest: '{{ httpd_config_parts_dir }}/15-graphite-web.conf'
+    regexp: "Require local$"
+    replace: "Require all granted"
   notify: restart httpd
 
 - name: listen on port 8080
   lineinfile:
-      dest: '{{ httpd_config_parts_dir }}/15-graphite-web.conf'
-      create: yes
-      state: present
-      insertafter: "# Graphite Web Basic mod_wsgi vhost"
-      line: "Listen {{ graphite_listen_port }}"
+    dest: '{{ httpd_config_parts_dir }}/15-graphite-web.conf'
+    create: yes
+    state: present
+    insertafter: "# Graphite Web Basic mod_wsgi vhost"
+    line: "Listen {{ graphite_listen_port }}"
```

Comment:
You may want to make this conditional...if someone has specified `graphite_listen_port == 80`, then the `Listen` statement here would conflict with the default `Listen` statement.